eth/executionclient/execution_client.go

```diff
@@ -41,7 +41,7 @@ type Provider interface {
 type SingleClientProvider interface {
 	Provider
 	SyncProgress(ctx context.Context) (*ethereum.SyncProgress, error)
-	streamLogsToChan(ctx context.Context, logs chan<- BlockLogs, fromBlock uint64) (lastBlock uint64, err error)
+	streamLogsToChan(ctx context.Context, logs chan<- BlockLogs, fromBlock uint64) (nextBlockToProcess uint64, err error)
```

Comment:
@nkryuchkov, maybe it's a good time to update the channel name to be more in line with the `go`? This is a broad topic with plenty of room for flexibility, so please consider this just as a suggestion!
https://groups.google.com/g/golang-nuts/c/NnZ-CuQAiT4